knowledge-base/troubleshooting-js-errors-maximum-call.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Causes and Solution for Maximum Call Stack Size Exceeded Error
+description: Learn what are the common reasons to get the Maximum call stack size exceeded error and how to fix it
+type: troubleshooting
+page_title: Maximum Call Stack Size Exceeded Error Causes and Solution
+slug: troubleshooting-kb-js-errors-maximum-call
+tags: telerik, blazor, troubleshooting, js-errors, maximum call stack size exceeded
+ticketid: 1641599, 1638959, 1639186, 1638288, 1642561, 1645266, 1641021
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>UI for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+## Description
+
+This knowledge base article deals with the following issues:
+* Getting `Maximum call stack size exceeded` error.
+
+## Error Message
+
+>warning Microsoft.JSInterop.JSException: Maximum call stack size exceeded
```

Comment:
Check if the error message is different in different browsers. In this case, add the variations in this section.